spanner/ddl/ast.go

```diff
@@ -181,16 +248,25 @@ func (ct CreateTable) PrintCreateTable(config Config) string {
 	var col []string
 	var colComment []string
 	var keys []string
-	for i, cn := range ct.ColNames {
-		s, c := ct.ColDefs[cn].PrintColumnDef(config)
-		s = "\n    " + s
-		if i < len(ct.ColNames)-1 {
-			s += ","
-		} else {
-			s += " "
+	if config.TargetDb == "experimental_postgres" {
```

Comment:
Use TARGET_EXPERIMENTAL_POSTGRES constant?

Reply:
We couldn't use the constant before because of circular dependency. Have now moved all constants to a top-level common package and now have updated to use constant as well.